contracts/test/ETO/MockETOCommitment.sol

```diff
@@ -43,6 +43,15 @@ contract MockETOCommitment is
         }
     }
 
+    // convenience function for moving all timestampts towards the past
+    // such that the next state transition will occur in delta seconds
+    function _shiftToBeforeNextState(uint8 delta) public {
+        require(delta < 180);
```

Comment:
so 3 minutes is good for everyone? also for QA?

Reply:
1800 it is now, approved by @glupiawymowka